### ISSUE TITLE
Commit 83b60339 broke use of pbogut/fzf-mru.vim plugin

### ISSUE THREAD

**Issue by micheleCTDE** [junegunn/fzf#3929]
### Checklist

- [X] I have read through the manual page (`man fzf`)
- [X] I have searched through the existing issues
- [X] For bug reports, I have checked if the bug is reproducible in the latest version of fzf

### Output of `fzf --version`

0.46.1 (debian)

### OS

- [X] Linux
- [ ] macOS
- [ ] Windows
- [ ] Etc.

### Shell

- [X] bash
- [ ] zsh
- [ ] fish

### Problem / Steps to reproduce

Running nvim v0.10.0 in debian testing (trixie), using the following plugins for fzf (Packer's syntaz):
```
  use { 'junegunn/fzf', run = 'fzf#install()'}
  use { 'ibhagwan/fzf-lua' }
  use 'pbogut/fzf-mru.vim'
```
Up to commit `01e76689`, running `:FZFMru` would open up the MRU file list thanks to the `fzf-mru` plugin.
Commit `83b60339` added a `--no-tmux` option to the `fzf` code and this is breaking the MRU plugin.
Error output:
```
Error running command cat '/tmp/nvim.user/nzXZJv/1'|'fzf'  --border  --prompt "MRU>"  --expect=ctrl-v,ctrl-x,ctrl-t --no-height --no-tmux > /tmp/nvim.user/nzXZJv/0
```

Running `fzf  --border  --prompt "MRU>"  --expect=ctrl-v,ctrl-x,ctrl-t --no-height --no-tmux` from CLI gives additional info: `unknown option: --no-tmux`.

Current workaround is to keep `fzf` to commit `01e76689`, but would be good to have a proper fix.

**Comment by junegunn** [comment]
You're running an outdated version of fzf that is not compatible with the latest Vim plugin you installed.
It's not clear how you ended up with the situation, because `fzf#install` should download the latest binary. 

Try manually calling it and see if it helps.

```vim
:call fzf#install()
:echo fzf#exec()
:echo system(fzf#exec() .. ' --version')
```

**Comment by micheleCTDE** [comment]
Hi @junegunn, thanks for the quick reply. 
Yes, it is an old `fzf` version, but it is what ships by default in `debian testing`. Any debian user using the version provided by the distribution would have a similar issue. Users of `debian stable` would have an even older version (0.38).
Given that debian is a well used distro, perhaps it would be good to have some conditional code and add the `--no-tmux` option only if `fzf` version is bigger than a specific value. What do you think?

**Comment by micheleCTDE** [comment]
I also need to correct myself. 
Version 0.46 is the `fzf` version used in my system OUTSIDE of `nvim`.
Inside `nvim`, `fzf` version `0.52.1 (6432f00)` is in use, as per output of `:echo system(fzf#exec() .. ' --version')`

Sorry for the misleading version information in first place.

**Comment by micheleCTDE** [comment]
I have now manually updated to version `0.54.0 (9e92b6f)`, somehow Packer was failing to update `fzf` automatically. The problem is gone with this version.
You can close the issue if you think it is not worth adding some conditional code as I mentioned above. Otherwise keep it open and close it after you have done that.
Thanks for your help, it pointed me in the right direction for further checks.

**Comment by junegunn** [comment]
Looks like you're misusing `run` directive. `fzf#install` is a Vim script expression not a shell command.

https://github.com/wbthomason/packer.nvim?tab=readme-ov-file#quickstart shows a `run` example that you can refer to.

**Comment by micheleCTDE** [comment]
You are right, thanks for highlighting that, as well for the amazing fzf :-)
